src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcessDumpUtility.cs

```diff
@@ -100,38 +105,63 @@ public void StartProcessDump(int processId, string dumpFileGuid, string testResu
         /// <returns>Arguments</returns>
         private static string BuildProcDumpArgs(int processId, string filename, bool isFullDump = false)
         {
+            ProcessDumpUtility.procDumpExceptionsList = new List<string>()
```

Comment:
Lets initialize this list at class level only. i.e. `private static List<string> procDumpExceptionsList;`, here rather than only defining, we can initialize it as well.